llvm/lib/Transforms/Utils/LCSSA.cpp

```diff
@@ -374,6 +388,8 @@ bool llvm::formLCSSA(Loop &L, const DominatorTree &DT, const LoopInfo *LI,
 
   SmallVector<BasicBlock *, 8> ExitBlocks;
   L.getExitBlocks(ExitBlocks);
+  if (!LoopExitBlocks.count(&L))
+    L.getExitBlocks(LoopExitBlocks[&L]);
```

Comment:
LoopExitBlocks.try_emplace(&L, ExitBlocks)

Reply:
The change I made here wasn't complete, I've fixed the code now.